src/agent/util/utils.ts

```diff
@@ -24,7 +24,10 @@ export const messages = {
   CAPTURE_BREAKPOINT_DATA: 'Error trying to capture snapshot data: ',
   INVALID_LINE_NUMBER: 'Invalid snapshot position: ',
   COULD_NOT_FIND_OUTPUT_FILE:
-      'Could not determine the output file associated with the transpiled input file'
+      'Could not determine the output file associated with the transpiled input file',
+  ASYNC_TRACES_WARNING: 'The Stackdriver debug agent does not use Inspector ' +
```

Comment:
nit: Maybe we should use:

`The Stackdriver Debugger for Node.js does not require V8 Inspector async stack traces. The INSPECTOR_ASYNC_STACK_TRACES_NOT_AVAILABLE can be ignored.
`

Reply:
Done.